src/amperity/gocd/agent/aurora/agent.clj

```diff
@@ -89,7 +89,7 @@
         :message "Agent tag must consist of lowercase letters"})
      (validate-float settings :cpu "cpu allocation" 0.1 32.0)
      (validate-int settings :ram "memory allocation" 256 16384)
-     (validate-int settings :disk "disk allocation" 256 16384)]))
+     (validate-int settings :disk "disk allocation" 256 65536)]))
```

Comment:
Hmm - I don't remember how I chose these values, you could probably crank both mem and disk even higher. 🤷‍♂